### ISSUE TITLE
grdimage OpenMP support on master branch

### ISSUE THREAD

**Issue by seisman** [GenericMappingTools/gmt#1951]
After merging 6.0 branch into master branch, the Windows CI build on master branch fails:
```
..\src\grdimage.c(1186): error C3016: 'row': index variable in OpenMP 'for' statement 
must have signed integral type
```

**Comment by joa-quim** [comment]
Yes, just noticed that too. Those lines are commented in 6.0 so, at least, the same should be donne on master.

**Comment by PaulWessel** [comment]
Yes, forgot to do that when this happened last week.  Please just comment out the OPenMP stuff for now in master as was done in 6.

**Comment by seisman** [comment]
Closed by #1953
